makefile

```diff
@@ -36,11 +36,6 @@ heroku-16-build:
 	@docker run -v $(shell pwd):/buildpack:ro --rm -it -e "STACK=heroku-16-build" heroku/heroku:16-build bash -c 'cp -r /buildpack /buildpack_test; cd /buildpack_test/; test/run;'
 	@echo ""
 
-cedar-14:
```

Comment:
I think the `test` target in this file needs updating to remove the `cedar-14` reference

Reply:
oh good catch! updated.